src/desktop-lite/README.md

```diff
@@ -48,7 +48,7 @@ You can also connect to the desktop using a [VNC viewer](https://www.realvnc.com
 
 ## Customizing Fluxbox
 
-The window manager is installed is [Fluxbox](http://fluxbox.org/). **Right-click** to see the application menu. In addition, any UI-based commands you execute inside the dev container will automatically appear on the desktop.
+The window manager installed is [Fluxbox](http://fluxbox.org/). **Right-click** to see the application menu. In addition, any UI-based commands you execute inside the dev container will automatically appear on the desktop.
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

Reply:
Thanks, sorry I didn't realise that.

I have made the change on my branch such that only NOTES.md has changes and force pushed, but this PR doesn't seem to have automatically updated as I'm used to seeing - I'm not sure why that is.

Reply:
Hmm, that's super strange! In another comment, I have added a suggestion (which should revert the change), does applying that help?

Reply:
Thanks - I committed the suggestion - apologies for the extra work.

Reply:
No worries, thanks!